flaskbb/templates/forum/forum.html

```diff
@@ -52,7 +52,7 @@
                 <div class="col-md-3 col-sm-3 col-xs-4 topic-last-post">{% trans %}Last Post{% endtrans %}</div>
             </div>
 
-            {% for topic, topicread in topics.items %}
+            {% for topic, last_post, topicread in topics.items %}
```

Comment:
Should I use ``last_post`` here instead of ``topic.last_post`` further down?

Reply:
I'd say use `last_post` in both places.